deploy/examples/cosi/bucketclass.yaml

```diff
@@ -2,7 +2,7 @@ kind: BucketClass
 apiVersion: objectstorage.k8s.io/v1alpha1
 metadata:
   name: sample-bcc
-driverName: ceph.objectstorage.k8s.io
+driverName: rook-ceph.ceph.objectstorage.k8s.io
```

Comment:
How will this change affect upgraded clusters?

Reply:
Yaa it will impact the upgraded clusters, even with a standalone ceph cosi driver as well. Please check @BlaineEXE comment https://github.com/ceph/ceph-cosi/pull/32#discussion_r1514839445 and make the ceph cosi driver mandate the driver prefix.

Reply:
I think that's fine. COSI is an alpha feature, so users should expect potential changes. We'd have to maintain code logic to check for this upgrade condition forever if we want to support no-impact upgrades.

Reply:
Does this mean existing COSI users will be broken on upgrade? Is there any workaround? We will at least need a note on this in the upgrade guide for 1.15. Even though it's officially alpha and rook still declares it as experimental, I'd suggest we document when there is a breaking change.

Reply:
Right. I forgot about existing buckets initially, but you are right. It is still best for users if we document in release notes when things break, and we should be able to document a procedure for continuing to work with existing storage wherever possible.

I don't think that we need to have an official doc that explains how to make the change, but it would be good to at least be able to link to something on github. Maybe a discussion post that describes the workflow needed to keep existing buckets working would be a good option.